src/main/resources/avro/reads.avdl

```diff
@@ -202,6 +207,12 @@ record ReadAlignment {
   The read ordinal in the fragment, 0-based and less than numberReads. This
   field replaces SAM flag 0x40 and 0x80 and is intended to more cleanly
   represent multiple reads per fragment.
+  A readNumber of 0 is equivalent to SAM flag 0x40 being set and 0x80 being
+  unset.
+  A readNumber of 1 is equivalent to SAM flag 0x80 being set and 0x40 being
+  unset.
+  A readNumber other than 0 or 1 means both SAM flags 0x40 and 0x80 are set.
```

Comment:
I _think_ that's what this PR is already saying. My naive understanding is that `readNumber` should correctly represent an arbitrary number of reads per fragment, as opposed to SAM's two-flag mechanism used to represent 0, 1 or many... (or should it be 1, 2 or many?)